modules/augain/augain.c

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file augain.c  Audio gain
+ *
+ * Copyright (C) 2024 Juha Heinanen
+ */
+
+#include <stdlib.h>
+#include <re.h>
+#include <rem.h>
+#include <baresip.h>
+
+
+/**
+ * Audio gain module
+ *
+ * This module can be used to increase volume of audio source,
+ * for example, microphone.
+ *
+ * Sample config:
+ *
+ \verbatim
+  augain            1.5
+ \endverbatim
+ */
+
+
+struct augain_enc {
+	struct aufilt_enc_st af;  /* base class */
+};
+
+
+static double gain = 1.0;
+
+
+static int conf_get_float(const struct conf *conf, const char *name,
```

Comment:
Should be removed after this is resolved: https://github.com/baresip/baresip/issues/3164

Reply:
Yes.